.goreleaser.yml

```diff
@@ -17,7 +17,7 @@ builds:
       post:
         # This will check plugin compatibility against latest version of Packer
         - cmd: |
-            go install github.com/hashicorp/packer/cmd/packer-plugins-check &&
+            go install github.com/hashicorp/packer/cmd/packer-plugins-check@latest &&
```

Comment:
so nice !

Reply:
no risking overriding dependencies anymore 👏🏼

Reply:
Would it be possible to require the version from the go.mod here instead ? To make sure everything's on par ?

Reply:
This is the version of Packer, which is not in `go.mod`. 🤔  It will always check against the latest version of packer.

Reply:
I'm a bit worried this will cause issues down the line, for example when we release a new api version and plugin maintainers want to test against the version of Packer their plugin refers to.

Reply:
We can read from a file. If they wish to release for a different packer version than the latest, they can update the file. What do you think?

Reply:
something like .packer_version

Reply:
🤔  good question, I'm trying to see if there's a go command we can use.